src/stacked_borrows.rs

```diff
@@ -587,10 +597,10 @@ trait EvalContextPrivExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
             // We immediately share it, to allow read accesses
             let two_phase_time = this.machine.stacked_borrows.increment_clock();
             let two_phase_bor = Borrow::Alias(Some(two_phase_time));
-            this.reborrow(new_place, size, /*fn_barrier*/false, two_phase_bor)?;
+            this.reborrow(new_place, size, false /* fn_barrier */, two_phase_bor)?;
```

Comment:
The name should be before the value, just like in a struct literal (`field_name: val`).